src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.251.1.mdx

```diff
@@ -0,0 +1,26 @@
+---
+subject: Browser agent
+releaseDate: "2024-01-29"
+version: 1.251.1
+features: []
+bugs: ["Pass unload options to simultaneous harvests in Session Replay","Fix deferred Session Replay payloads"]
+security: []
+---
+
+## v1.251.1
+
+### Bug fixes
+
+#### Pass unload options to simultaneous harvests in Session Replay
+Pass unload options to simultaneous harvests in Session Replay to allow all calls to utilize sendBeacon when unloading
+
+#### Fix deferred Session Replay payloads
+Force agent to send preloaded session replay payloads if present at load time.
+
+## Support statement:
+
+New Relic recommends that you upgrade the agent regularly to ensure that you're getting the latest features and performance benefits. Older releases will no longer be supported when they reach [end-of-life](https://docs.newrelic.com/docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy/). Release dates are reflective of the original publish date of the agent version.
+
+New Browser Agent releases are rolled out to customers in small stages over a period of time. Because of this, the date the release becomes accessible to your account may not match the original publish date. Please see this [status dashboard](https://newrelic.github.io/newrelic-browser-agent-release/) for more information.
```

Comment:
```suggestion
New browser agent releases are rolled out to customers in small stages over a period of time. Because of this, the date the release becomes accessible to your account may not match the original publish date. Please see this [status dashboard](https://newrelic.github.io/newrelic-browser-agent-release/) for more information.
```